### PR TITLE
add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kilograham**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
 
-Open PRs against the [develop](https://github.com/raspberrypi/pico-examples/tree/develop) branch
+**NOTE:** Development takes place on the `develop` branch in this repository. Please open your https://github.com/raspberrypi/pico-examples/pulls[pull request] (PR) against the [`develop`](https://github.com/raspberrypi/pico-examples/tree/develop) branch, pull requests against the `master` branch will automatically CI fail checks and will not be accepted. You will be asked to rebase your PR against `develop` and if you do not do so, your PR will be closed.
 
-### Any contributions you make will be under the BSD-3 Software License
-In short, when you submit code changes, your submissions are understood to be under the same [BSD-3 License](LICENSE.TXT) that covers the project.
+While we are happy to take contributions, big or small, changes in the SDK may have knock-on effects in other places so it is possible that apparently benign pull requests that make seemingly small changes could be refused.
```
[the remark this answers is not in the document]
Comment (lurch) [reply CONTRIBUTING.md:13]: (or at the very least, "the SDK" needs changing to "the Examples")

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
-## Contributing code / examples
+In order to contribute new or updated code, you must first create a GitHub account and fork the original repository to your own account. You can make changes, save them in your repository, then [make a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) against this repository. The pull request will appear [in the repository](https://github.com/raspberrypi/pico-examples/pulls) where it can be assessed by the maintainers, and if appropriate, merged with the official repository.
 
-Open PRs against the [develop](https://github.com/raspberrypi/pico-examples/tree/develop) branch
+**NOTE:** Development takes place on the `develop` branch in this repository. Please open your https://github.com/raspberrypi/pico-examples/pulls[pull request] (PR) against the [`develop`](https://github.com/raspberrypi/pico-examples/tree/develop) branch, pull requests against the `master` branch will automatically CI fail checks and will not be accepted. You will be asked to rebase your PR against `develop` and if you do not do so, your PR will be closed.
```
[the remark this answers is not in the document]
Comment (kilograham) [reply CONTRIBUTING.md:11]: ok, but it should be

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+**NOTE:** Development takes place on the `develop` branch in this repository. Please open your https://github.com/raspberrypi/pico-examples/pulls[pull request] (PR) against the [`develop`](https://github.com/raspberrypi/pico-examples/tree/develop) branch, pull requests against the `master` branch will automatically CI fail checks and will not be accepted. You will be asked to rebase your PR against `develop` and if you do not do so, your PR will be closed.
+
+While we are happy to take contributions, big or small, changes in the SDK may have knock-on effects in other places so it is possible that apparently benign pull requests that make seemingly small changes could be refused.
```
Comment (lurch) [inline CONTRIBUTING.md:13]: Still says "the SDK" here.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kilograham) [reply CONTRIBUTING.md:13]: weird github wasnt showing me all the comments yesterday

**Review by lurch — APPROVED** [review]
